xlators/protocol/server/src/server.c

```diff
@@ -685,10 +685,8 @@ server_graph_janitor_threads(void *data)
         LOCK(&ctx->volfile_lock);
         {
```

Comment:
I think we don't even need this variable and avoid next 3 lines.
We can only use destroy_ctx. Use top->children on line number 690, update destroy_ctx which can be used on 702

Reply:
Okay, ig you mean like this:

```
LOCK(&ctx->volfile_lock);
        {
            if (!top->children && !ctx->destroy_ctx) {
                ctx->destroy_ctx = _gf_true;
                destroy_ctx = _gf_true;
            }
        }
UNLOCK(&ctx->volfile_lock);
out:
    free(arg->victim_name);
    free(arg);
    if (!top->children && destroy_ctx) {
        gf_log(.......
```